### PR TITLE
693 change dns to station.massa and paths URL

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: pkg/config/network.go [hunk 1]
```diff
@@ -67,3 +69,7 @@ func GetDNSAddress(urlOrNetwork string, dnsFlag string) string {
 		return ""
 	}
 }
+
+func GetMassaStationURL() string {
```
Comment (peterjah) [inline pkg/config/network.go:73]: useless Getter, just put a uppercase to `massaStationURL`

File: int/systray/systray.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ func MakeGUI(logo []byte) (fyne.App, *fyne.Menu) {
 		}
 
 		homeShortCutMenu.Action = func() {
-			openURL(&stationGUI, "http://my.massa/thyra/home")
+			openURL(&stationGUI, "http://station.massa/thyra/home")
```
Comment (peterjah) [inline int/systray/systray.go:44]: use massaStationURL variable from network.go

File: pkg/node/sendoperation/sendoperation.go [hunk 3]
```diff
@@ -28,7 +28,7 @@ const HundredMassa = 100_000_000_000
 
 const OneMassa = 1_000_000_000
 
-const WalletPluginURL = "http://my.massa/thyra/plugin/massalabs/wallet/rest/wallet/"
+const WalletPluginURL = "http://station.massa/thyra/plugin/massalabs/wallet/rest/wallet/"
```
Comment (peterjah) [inline pkg/node/sendoperation/sendoperation.go:31]: use massaStationURL variable from network.go

**Review by peterjah — CHANGES_REQUESTED** [review]

**Review by gregLibert — COMMENTED** [review]
This PR is big by nature. But it contains modifications having nothing to do with the renaming task...
All is fine, except for the postman question, maybe we should simply remove them. but this is not blocking

File: api/test/massa_endpoints.postman_collection.json [hunk 4]
```diff
@@ -42,7 +42,7 @@
 				"method": "GET",
 				"header": [],
 				"url": {
-					"raw": "http://my.massa/massa/addresses?addresses=AU12ajG8wNxcdv7V7wEssGfQSaU5YX6oQQ9Sj686vukvfp2LuUGz5&addresses=AU12tkFEY2WTfHA4H8F5wPWp4JNKCBNyq5AhLdZj7udfBYoRtp1qS",
+					"raw": "http://station.massa/massa/addresses?addresses=AU12ajG8wNxcdv7V7wEssGfQSaU5YX6oQQ9Sj686vukvfp2LuUGz5&addresses=AU12tkFEY2WTfHA4H8F5wPWp4JNKCBNyq5AhLdZj7udfBYoRtp1qS",
```
[the remark this answers is not in the document]
Comment (Thykof) [reply api/test/massa_endpoints.postman_collection.json:45]: no, not yet, and we have in mind to generate the postman collection automatically from the swagger file. 

the changes here is simple: replace my.massa by station.massa.

**Comment by Thykof** [comment]
The CI is red because of the test-api-robot that doesn't pass. The fix is in the following PR: #724

**Review by gregLibert — APPROVED** [review]

**Pull request merged**